src/raft_server.c

```diff
@@ -537,10 +563,14 @@ static int __should_grant_vote(raft_server_private_t* me, msg_requestvote_t* vr)
     if (my_node && !raft_node_is_voting(my_node))
         return 0;
 
+    /* For a prevote, we could proceed to the votedFor check, if
```

Comment:
is the comment saying that we could theoretically proceed to the votedFor check in this scenario (but we are choosing not to)? Just making sure I am reading the comment correctly.